src/components/FindPark.vue

```diff
@@ -13,12 +13,16 @@
         Find a dog park near me!
       </button>
     </router-link>
-    <h3>--Or--</h3>
+    <h3 id="location-message" v-if="!this.$store.state.geolocation">
```

Comment:
Does this message show up even if the user has location services turned on and they're waiting for the app to retrieve their coordinates?  If not, this might lead the user to go into their settings before they truly have to.  Maybe we want a `setTimeout` to handle this...something like "Grabbing your location..." and then after 5 seconds, show the message about turning on location access and reloading?

Reply:
This was hard for me to assess given the other location stuff going on my computer that seems to be different than yours. When I refresh the page, there is about a 4second lapse until the button becomes active and, yes, the message is displayed during that time. I think either a message with a setTimeout or Nathan's loading animation might be useful if it is happening across the board.

Reply:
Makes sense to me! Maybe we can take a look at that tomorrow morning.